modules/ztp-du-host-bios-requirements.adoc

```diff
@@ -0,0 +1,75 @@
+// CNF-1500 ZTP
+// Module included in the following assemblies:
+//
+// scalability_and_performance/ztp-zero-touch-provisioning.adoc
+
+[id="configuring_bios_for_du_hosts_{context}"]
+= Configuring BIOS for distributed unit bare-metal hosts
+
+Distributed unit (DU) hosts require certain BIOS settings to be configured before the host can be provisioned.
+
+.Procedure
+. Set the *UEFI/BIOS Boot Mode* to `UEFI` and ensure that the same setting is applied in the `BareMetalHost` CRD.
+
+. Set *Hard drive first* in the host boot sequence order.
++
+[NOTE]
+====
+If `rootDeviceHints` is specified in the `BareMetalHost` CRD, the hard disk must match the hard disk that you configure in the host boot sequence order.
```

Comment:
> the hard disk must match

Is this enough info for the user to match this up? I suggest considering adding a syntax example later.